front_end/src/app/admin/events/manage-children/manage-children.component.html

```diff
@@ -35,85 +35,27 @@
             </tr>
           </thead>
           <tbody>
-            <tr>
-              <td>
-                Andrea
-              </td>
-              <td>
-                Smith
-              </td>
-              <td>
-
-              </td>
-              <td>
-                1234
-              </td>
-              <td>
-                Nursery A K112
-              </td>
-              <td>
-                4:15pm
-              </td>
-              <td>
-                Bob Smith, Dee Dee Smith
-              </td>
-              <td>
-                <a class="btn btn-primary btn-sm" href="#">Reverse</a>
-              </td>
-              <td>
-                <a class="btn btn-primary btn-sm" href="#">Reprint</a>
-              </td>
-            </tr>
-            <tr>
-              <td>
-                Andrea
-              </td>
-              <td>
-                Smith
-              </td>
-              <td>
-                <span class="tag tag-pill tag-default">Guest</span>
-              </td>
-              <td>
-                1234
-              </td>
-              <td>
-                Nursery A K112
-              </td>
-              <td>
-                4:15pm
-              </td>
-              <td>
-                Bob Smith, Dee Dee Smith
-              </td>
-              <td>
-                <a class="btn btn-primary btn-sm" href="#">Reverse</a>
-              </td>
-              <td>
-                <a class="btn btn-primary btn-sm" href="#">Reprint</a>
-              </td>
-            </tr>
-            <tr>
+            <tr *ngFor="let child of children">
               <td>
-                Andrea
+                {{child.FirstName}}
               </td>
               <td>
-                Smith
+                {{child.LastName}}
               </td>
               <td>
```

Comment:
the guest badge should be able to be put in here now. We could see if their HouseholdId is the scripted HouseholdId for guests?